lambda_functions/ml_service/template.yaml

```diff
@@ -159,6 +159,46 @@ Resources:
           - LambdaExecutionRole
           - Arn
 
+  GetTagsFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub "detektivkollektiv-ml_api-GetTags-${STAGE}"
+      CodeUri: .
+      Handler: GetEntities.get_tags
+      Runtime: python3.8
+      MemorySize: 832
+      Role:
+        Fn::GetAtt:
+          - LambdaExecutionRole
+          - Arn
+
+  GetTagsByItemidFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      FunctionName: !Sub "detektivkollektiv-ml_api-get_tags_by_itemid-${STAGE}"
+      CodeUri: .
+      Handler: GetTags.get_tags_for_item
+      Runtime: python3.8
+      MemorySize: 832
+      Role:
+        Fn::GetAtt:
+          - LambdaExecutionRole
+          - Arn
+      Events:
+        Echo:
+          Type: Api # More info about API Event Source: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#api
+          Properties:
+            Path: /factchecks/{item_id}
```

Comment:
URL path does not seem to be correct.

Something like /tags?item_id=<item_id> seems more fitting.
item_id has then to be read from queryPathParameters obviously.

Reply:
Ups ;-)